cluster/gce/util.sh

```diff
@@ -2545,17 +2570,27 @@ function create-master() {
   KUBERNETES_MASTER_NAME="${MASTER_RESERVED_IP}"
   MASTER_ADVERTISE_ADDRESS="${MASTER_RESERVED_IP}"
 
-  create-certs "${MASTER_RESERVED_IP}"
+  MASTER_INTERNAL_IP=""
+  if [[ ${GCE_PRIVATE_CLUSTER:-} == "true" ]]; then
+    gcloud compute addresses create "${MASTER_NAME}-internal-ip" --project "${PROJECT}" --region $REGION --subnet $SUBNETWORK
```

Comment:
Do you know why for static IP we have to wait and here we don't:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L1817
?

Reply:
Good question. I tried to understand context of retries in create-static-ip and I think there are two parts:
* https://github.com/kubernetes/kubernetes/pull/15252 which adds some retries as apparently the compute addresses create operation has failed making it impossible to boot the cluster
* https://github.com/kubernetes/kubernetes/pull/33097 which adds some waiting after gcloud compute addresses create for gcloud compute addresses describe finishes.

While I see that significant part of gcloud invocations is in retry loop, there are some without and I have never seen any issue with them (e.g. all calls in create-network).

I think it's reasonable to start with no retries and add some retry logic if we find that this is flaky. WDYT?